pkg/engine/anchor/anchor_test.go

```diff
@@ -0,0 +1,653 @@
+package anchor
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestNew(t *testing.T) {
+	type args struct {
+		modifier AnchorType
+		key      string
+	}
+	tests := []struct {
+		name string
```

Comment:
`name` is not used? Same for all the following tests.

Reply:
Was too lazy to set names everywhere.

Reply:
Indexing them is good enough for us to identify failure tests IMO:)